sklearn/feature_selection/_base.py

```diff
@@ -121,3 +124,47 @@ def inverse_transform(self, X):
         Xt = np.zeros((X.shape[0], support.size), dtype=X.dtype)
         Xt[:, support] = X
         return Xt
+
+
+def _get_feature_importances(estimator, getter, transform_func=None,
+                             norm_order=1):
+    """Retrieve or aggregate feature importances from estimator"""
```

Comment:
It could be worth to add a docstring to clarify exactly what it does:

```python
Retrieve or aggregate and optionally transform the feature importances
from an estimator

Parameters
----------
estimator : estimator
    A scikit-learn estimator from which we want to get the feature importances.

getter : "auto", str or callable
    An attribute or a callable to get the feature importance. If `"auto"`, `estimator`
    is expected to expose `coef_` or `feature_importances`.

transform_func : {"norm", "square"}, default=None
    The transform to apply to the feature importances. By default (`None`)
    no transformation is applied.

norm_order : int, default=1
    The norm order to apply when `transform_func="norm"`. Only applied
    when `importances.ndim > 1`.

Returns
-------
importances : ndarray of shape (n_features,)
    The features importances, optionally transformed.
```